### ISSUE TITLE
ML Filter FORCE_FSM_IN error

### ISSUE THREAD

**Issue by jchang97** [ConesaLab/SQANTI3#168]
Hi there,

SQANTI v5.1 ML works fine with default parameters, but when I include -f FORCE_FSM_IN, it generates this error for me (as below). The classification.txt files and/or randomforest.RData objects have also been generated with v5.1.
```

â„¹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Error in `dplyr::mutate()`:
! Problem while computing `structural_category = ... %>% ...`.
Caused by error in `factor()`:
! object 'structural_category' not found
Backtrace:

```
Could I please have some guidance on this?

Thank you very much!

**Comment by aarzalluz** [comment]
Hi @jchang97 -Thank you for reporting this. I will test it out to see if I can reproduce your error and get back to you as soon as I can.

**Comment by aarzalluz** [comment]
Hi @jchang97,

Sorry for the late reply. I have not been able to replicate your error -the SQ3 ML filter runs perfectly on the example dataset when using the -f option.

Even though you did not provide the exact command you used to run the filter, I did notice that you said that you included `-f FORCE_FSM_IN`. However, the correct way to supply the argument would be  `-f TRUE`, whereas by default the filter uses `-f FALSE`. I have updated the documentation so that it does not lead to confusion anymore.

Hope that helps,

Ángeles
